src/EventService.Data.Provider.MsSql.Ef/Migrations/20230126193800_InitialTables.cs

```diff
@@ -157,6 +157,85 @@ private void CreateEventFilesTable(MigrationBuilder migrationBuilder)
       });
   }
   
+  private void CreateGroupsTable(MigrationBuilder migrationBuilder)
+  {
+    migrationBuilder.CreateTable(
+      name: DbGroup.TableName,
+      columns: table => new
+      {
+        Id = table.Column<Guid>(nullable: false),
+        Name = table.Column<string>(nullable: false),
+        MaxGuests = table.Column<int>(nullable: true),

```

Comment:
not sure if maxGuests should be nullable. That could be a problem during distribution